haystack/cli/README.md

```diff
@@ -13,32 +13,42 @@ This is the repository where we keep the code for the Haystack CLI.
 
 To contribute to the tutorials please check out our [Contributing Guidelines](./Contributing.md)
 
-## Available commands
+## Available CLI commands
 
 ### `haystack prompt fetch`
 
-```
-Usage: haystack prompt fetch [OPTIONS] [PROMPT_NAME]...
+**Usage:**
 
-  Downloads a prompt from the official Haystack PromptHub and saves it locally
-  to ease use in environments with no network.
+```
+haystack prompt fetch [OPTIONS] [PROMPT_NAME]
+```
 
-  PROMPT_NAME can be specified multiple times.
+Download a prompt from the official [Haystack PromptHub](https://prompthub.deepset.ai/) and save it locally
+for easier use in environments with no network.
 
-  PROMPTHUB_CACHE_PATH environment variable can be set to change the default
-  folder in which the prompts will be saved in.
+- PROMPT_NAME can be specified multiple times.
 
-  If a custom PROMPTHUB_CACHE_PATH is used, remember to also use it for
-  Haystack invocations.
+- PROMPTHUB_CACHE_PATH environment variable can be set to change the default
+folder in which the prompts will be saved in. You can find the default cache path on your machine by running the following code:
 
-  The Haystack PromptHub is https://prompthub.deepset.ai/
+  ``` python
+  from platformdirs import user_data_dir
+  from pathlib import Path
+  Path(user_data_dir("haystack", "deepset")) / "prompthub_cache"
```

Comment:
I think it's better to suggest:

```suggestion
  from haystack.nodes.prompt.prompt_template import PROMPTHUB_CACHE_PATH
  print(PROMPTHUB_CACHE_PATH)
```

Reply:
@ZanSara Is adding a command to the CLI to print `PROMPTHUB_CACHE_PATH` counterintuitive? This way, users can run a command and learn what their default `PROMPTHUB_CACHE_PATH` is.

Reply:
That's a great idea! Let's do that

Reply:
I added Sara's suggested command in the meantime, but will change to cli command once we have that!